_includes/cockroachcloud/cockroachcloud-ask-admin.md

```diff
@@ -1,3 +1,3 @@
 {{site.data.alerts.callout_info}}
-Only [Console Admins](console-access-management.html#console-admin) can create SQL users. If you are a [Developer](console-access-management.html#developer), you need to ask your Console Admin for the credentials of a SQL user to access the cluster. To find out who your Console Admin is, [log in](https://cockroachlabs.cloud/) and navigate to **Cluster Overview** > **Access**.
```

Comment:
Hmm, good point. That was old copy. Is there a way to find out who your Admin is?
If you are just an org member, can you see the /access page and see who has what roles so you can identify an admin?

Reply:
If you're an org member, there isn't a way right now. For others, they can go to the access page. I believe we're planning on opening the `/access` page up to org members in phase 2 so that everyone can see the admins, but that's still tbd.

Reply:
so if you are an org member, is there any way to 'find out who your org admin is'? Does it say in your invitation to the org who invited you?